packages/lib-utils/src/k8s/hooks/k8s-watcher.ts

```diff
@@ -77,14 +77,14 @@ export const getReduxData = (immutableData, resource: WatchK8sResource) => {
 };
 
 export const getWatchData: GetWatchData = (resource, k8sModel, cluster) => {
-  if (!k8sModel || !resource || !resource.name || !resource.namespace) {
+  if (!k8sModel || !resource || !k8sModel.name || !resource.namespace) {
     return null;
   }
   const query = makeQuery(
     resource.namespace,
     resource.selector,
     resource.fieldSelector,
-    resource.name,
+    k8sModel.name,
```

Comment:
Why are we replacing `resource.name` with `k8sModel.name` here? Since resource.name corresponds to the name of the specific watched resource which we pass to the hook.

Reply:
When we are getting list the name is not required and line 80 was failing. Since the name is not required for list methods I'll change it so the `if` is more aligned. Goot poin however!